torch/csrc/jit/serialization/python_print.cpp

```diff
@@ -864,6 +864,7 @@ struct PythonPrintImpl {
 
   void printConstant(TaggedStringStream& stmt, const IValue& v) {
     const auto customFormatter = [&](std::ostream& ss, const IValue& v) {
+      // TODO Add support for serialization of custom classes
```

Comment:
Should there be an assertion here? 
To make sure serialization is not invoked where there is a custom class as a constant